website/docs/cli/commands/describe/describe-component.md

```diff
@@ -294,8 +344,34 @@ component_info:
           line: 23
     diagnostics: []
 deps:
+  - catalog/terraform/mixins/test-2
```

Comment:
This example seems overwhelmingly large and has no annotations to explain what's going on.